client/api_group.go

```diff
@@ -91,23 +91,25 @@ func (c *client) UpdateGroupMember(ctx context.Context, groupName string, groupO
 	if groupName == "" {
 		return "", errors.New("group name is empty")
 	}
-
 	if len(addAddresses) == 0 && len(removeAddresses) == 0 {
 		return "", errors.New("no update member")
 	}
-
-	addMembers := make([]*storageTypes.MsgGroupMember, 0)
-	removeMembers := make([]sdk.AccAddress, 0)
-
-	if len(addAddresses) != len(expirationTime) {
+	if opts.ExpirationTime != nil && len(addAddresses) != len(opts.ExpirationTime) {
 		return "", errors.New("please provide expirationTime for every new add member")
 	}
-
+	addMembers := make([]*storageTypes.MsgGroupMember, 0)
+	removeMembers := make([]sdk.AccAddress, 0)
+	expirationTime := make([]*time.Time, len(addAddresses))
 	for idx, addr := range addAddresses {
 		_, err := sdk.AccAddressFromHexUnsafe(addr)
 		if err != nil {
 			return "", err
 		}
+		if opts.ExpirationTime != nil && opts.ExpirationTime[idx] != nil {
+			expirationTime[idx] = opts.ExpirationTime[idx]
+		} else {
+			expirationTime[idx] = &storageTypes.MaxTimeStamp
+		}
```

Comment:
ExpirationTime in MsgGroupMember should accpet time.Time,  but here pass the *time.Time

Reply:
it is also pointer in greenfield